js/README.md

```diff
@@ -43,3 +43,27 @@ There are several options that can be passed the function in the optional
   detected. This is intended to enrich the module graph with external types that
   are resolved in some other fashion, like from a `package.json` or via
   detecting an `@types` typings package is available.
+
+## Usage with `import_map`
+
+You can use [`import_map`](https://deno.land/x/import_map) in combination with
+`deno_graph` in order to fully resolve the specifiers in your graph.
+
+```ts
+import { createGraph } from "https://deno.land/x/deno_graph@0.63.6/mod.ts";
+import { resolve, toFileUrl } from "https://deno.land/std@0.212.0/path/mod.ts";
+import { parseFromJson } from "https://deno.land/x/import_map@v0.18.3/mod.ts";
+
+const path = resolve("./entrypoint.ts");
+const importMap = Deno.readTextFileSync(
+  new URL("./deno.json", import.meta.url),
+);
+const resolvedImportMap = await parseFromJson(
+  toFileUrl(resolve("deno.json")),
+  importMap,
+);
+const graph = await createGraph(toFileUrl(path).href, {
+  resolve: resolvedImportMap.resolve.bind(resolvedImportMap),
+});
+console.log(graph);
+```
```

Comment:
Could you add comments here? The code is relatively clear but I wish it was more clearly explained.

Reply:
I added some comments. Let me know what you think.